pkg/api/types.go

```diff
@@ -921,6 +921,9 @@ type PodSpec struct {
 	// used must be specified.
 	// Optional: Default to false.
 	HostNetwork bool `json:"hostNetwork,omitempty"`
+	// Run the container using the host ipc mode.
+	// Optional: Default to false.
+	HostIPC bool `json:"hostIPC,omitempty"`
```

Comment:
Our intent is to make "pod namespace IPC" the default once that is supported by Docker.